.github/workflows/ci.yml

```diff
@@ -39,11 +39,19 @@ jobs:
           turbo-team: ${{ vars.TURBO_TEAM }}
           turbo-token: ${{ secrets.TURBO_TOKEN }}
 
+      - name: List node_modules
+        run: npm ls
+        shell: bash
```

Comment:
Can we remove this?

Reply:
We could though it errorred when the issues first occurred and could be a good litmus check.

Reply:
👍 let's move it to init then, wdyt?

Reply:
I'll do it in a follow-up